browser-test/src/application_navigation.test.ts

```diff
@@ -502,6 +506,109 @@ describe('Applicant navigation flow', () => {
       await applicantQuestions.clickSubmit()
       await applicantQuestions.expectIneligiblePage()
     })
+
+    it('shows not eligible upon submit with ineligible answer with gating eligibility', async () => {
```

Comment:
**suggestion:** Maybe say something like "with gating eligibility available, but not set" to make this a bit more clear?

Reply:
To clarify, eligibilityIsGating is by default true and means that eligibility does gate submission. If it's set to false, then it does NOT gate submission. So, I don't think your suggestion is accurate but I could change it to 'shows not eligible upon submit with ineligible answer with non-gating eligibility available, but not set' if you think that's clearer, or maybe just 'with eligibility set to gating' is clearer?